ansible/playbook.yml

```diff
@@ -34,25 +34,66 @@
       - wget
       - jq
       state: present
+  
+  - name: Check if cluster exists
+    stat:
+      path: "{{ansible_user_dir}}/ansible"
+    register: ocp_cluster
 
-  - name: Save pull secret file
-    ansible.builtin.copy:
-      src: "{{ pull_secret_file }}"
-      dest: "{{ pull_secret_remote_path }}"
+  - name: Clean OCP cluster if exists
```

Comment:
You can merge these two tasks into one block (https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_blocks.html). This way you can check `ocp_cluster.stat.exists` only once

Reply:
Nice, thanks for the tip .... changed it